src/requests/textdocument.jl

```diff
@@ -8,22 +8,17 @@ function process(r::JSONRPC.Request{Val{Symbol("textDocument/didOpen")},DidOpenT
         doc._version = r.params.textDocument.version
         get_line_offsets(doc, true)
```

Comment:
This branch needs a `set_open_in_editor(doc, true)`.

Also, doesn't this branch also need a call to `try_to_load_parents`?